server/command.go

```diff
@@ -96,22 +96,7 @@ func executeConnect(p *Plugin, c *plugin.Context, header *model.CommandArgs, arg
 		return p.help(header)
 	}
 
-	instance, err := p.currentInstanceStore.LoadCurrentJIRAInstance()
-	if err != nil {
-		return p.responsef(header, "There is no Jira instance installed. Please contact your system administrator.")
-	}
-
-	jiraUser, err := p.userStore.LoadJIRAUser(instance, header.UserId)
-	if err == nil && len(jiraUser.Key()) != 0 {
-		return p.responsef(header, "You already have a Jira account linked to your Mattermost account. Please use `/jira disconnect` to disconnect.")
-	}
-
-	redirectURL, err := instance.GetUserConnectURL(header.UserId)
-	if err != nil {
-		return p.responsef(header, "Command failed, please contact your system administrator: %v", err)
-	}
-
-	return p.responseRedirect(redirectURL)
+	return p.responsef(header, "Please connect your account using the desktop or browser client.")
```

Comment:
I took out the bits checking for an existing instance / connected account because it is irrelevant in this context.